core/sdk/src/main/kotlin/com/viam/sdk/core/component/audioinput/AudioInput.kt

```diff
@@ -0,0 +1,55 @@
+package com.viam.sdk.core.component.audioinput
+
+import com.viam.common.v1.Common.ResourceName
+import com.viam.component.audioinput.v1.Audioinput.ChunksResponse
+import com.viam.component.audioinput.v1.Audioinput.PropertiesResponse
+import com.viam.sdk.core.component.Component
+import com.viam.sdk.core.resource.Resource
+import com.viam.sdk.core.resource.Subtype
+import com.viam.sdk.core.robot.RobotClient
+
+typealias AudioStream = Iterator<ChunksResponse>
+typealias Properties = PropertiesResponse
```

Comment:
Do kotlin typealiases work in java?

Reply:
dont think they are visible in Java but [this website](https://typealias.com/guides/all-about-type-aliases/#:~:text=Java%20Interoperability) says Java will just work with the underlying type